packages/applications-service-api/src/utils/map-applications-to-csv.js

```diff
@@ -1,9 +1,37 @@
 const { stringify } = require('csv-stringify/sync');
 const moment = require('moment');
-const { mapColumnLabelToApi } = require('./application.mapper');
+const { stageNameFromValue } = require('./application.mapper');
 
 const formatDate = (date) => (moment(date).isValid() ? moment(date).format('YYYY-MM-DD') : '');
 
+const formatStageForCSV = (application) => {
+	const isMaterialChange =
+		application.isMaterialChange && application.sourceSystem === 'BACK_OFFICE';
```

Comment:
Can we just confirm that the sourceSystem will be set to BACK_OFFICE ?  Just want to make sure we are not ignoring this unnecessarily.  I checked the API in dev for BC0110005, the api returns - "sourceSystem": "back-office-applications",

It may get mapped somewhere to 'BACK_OFFICE' but worth checking.

Reply:
Good spot, it looks like `back-office-applications` is the right value. `BACK_OFFICE` is used elsewhere.